.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,222 @@
+        > workflow][XKCD 1172].
+        > We try to be mindful about this.
+        >
+        > Also test if the latest release and main branch are affected too.
+
+        [issue search]: ../search?q=is%3Aissue&type=issues
+        [XKCD 1172]: https://xkcd.com/1172/
+
+  - type: textarea
+    attributes:
+      label: Summary
+      description: Explain the problem briefly below.
+      placeholder: >-
+        When I try to do X with `digitalocean.cloud` from the `main` branch on GitHub, Y
+        breaks in a way Z under the env E. Here are all the details I know
+        about this problem...
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Issue Type
+      description: >
+        Please select the single available option in the drop-down.
+
+        <details>
+          <summary>
+            <em>Why?</em>
+          </summary>
+
+          We would do it by ourselves but unfortunatelly, the curent
+          edition of GitHub Issue Forms Alpha does not support this yet 🤷
+
+          _We will make it easier in the future, once GitHub
+          supports dropdown defaults. Promise!_
+
+        </details>
+
+      options:
+        - Bug Report
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Component Name
+      description: >
+        Write the short name of the rst file, module, plugin, task or
+        feature below, *use your best guess if unsure*.
+
+      placeholder: droplet, one_click, domain_record, account_info, etc.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Ansible Version
+      description: >-
+        Paste verbatim output from `ansible --version` below, under
+        the prompt line. Please don't wrap it with triple backticks — your
+        whole input will be turned into a code snippet automatically.
+      render: console
+      value: |
+        $ ansible --version
+      placeholder: |
+        $ ansible --version
+        ansible [core 2.11.0b4.post0]  (detached HEAD ref: refs/) last updated 2021/04/02 00:33:35 (GMT +200)
+        config file = None
+        configured module search path = ['~/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
+        ansible python module location = ~/src/github/ansible/ansible/lib/ansible
+        ansible collection location = ~/.ansible/collections:/usr/share/ansible/collections
+        executable location = bin/ansible
+        python version = 3.9.0 (default, Oct 26 2020, 13:08:59) [GCC 10.2.0]
+        jinja version = 2.11.3
+        libyaml = True
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Configuration
+      description: >-
+        Paste verbatim output from `ansible-config dump --only-changed -t all` below, under the prompt line.
+        (if using a version older than ansible-core 2.12 you should omit the '-t all')
```

Comment:
We're not [testing < 2.14](https://github.com/digitalocean/ansible-collection?tab=readme-ov-file#tested-with-versions) and it's EOL, should we mention this at all?

Reply:
Thanks, I changed some lines from what ansible/ansible uses, but did copy some large chunks. I agree, we don't need to mention in our project.